mysql-test/suite/innodb/t/innodb_stats.test

```diff
@@ -63,3 +63,16 @@ CREATE TABLE test_innodb_stats (
 -- disable_query_log
 DROP TABLE test_innodb_stats;
 set @@use_stat_tables= @save_use_stat_tables;
+--enable_query_log
+--echo #
+--echo #  MDEV-31740 InnoDB statistics column length validation failed
+--echo #
+call mtr.add_suppression("InnoDB: Table mysql.innodb_table_stats has length mismatch in the column name last_update. Please either run 'mariadb-upgrade' or fix it manually with: ALTER TABLE mysql.innodb_table_stats MODIFY last_update");
```

Comment:
We want to match literal `.`, not any character. Does the suppression have to include the whole message? Could we please truncate it before the "Please" part?